docs/maui/views/MediaElement.md

```diff
@@ -77,11 +77,65 @@ public class MainActivity : MauiAppCompatActivity
 }
 ```
 
+Edit the `androidManifest.xml` manifest file and add the following.
+
+```csharp
+<application android:allowBackup="true" android:icon="@mipmap/appicon" android:enableOnBackInvokedCallback="true" android:supportsRtl="true">
+    <service android:name="CommunityToolkit.Maui.Media.Services" android:exported="false" android:enabled="true" android:foregroundServiceType="mediaPlayback">
+      <intent-filter>
+        <action android:name="android.intent.action.MEDIA_BUTTON" />
+      </intent-filter>
+      <intent-filter>
+        <action android:name="androidx.media3.session.MediaSessionService"/>
+      </intent-filter>
+    </service>
+</application>
+<uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+<uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
+<uses-permission android:name="android.permission.FOREGROUND_SERVICE_MEDIA_PLAYBACK" />
+<uses-permission android:name="android.permission.MEDIA_CONTENT_CONTROL" />
+```
+
+> [!NOTE]
+> This modification to the Android manifest enables metadata display when playing a video. It provides support for notifications and is essential
+ for notifications to function across all relevant APIs. The change introduces a service and grants necessary permissions.
+
 For a full example of this method included in an application please refer to the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Platforms/Android/MainActivity.cs)
 
-### [iOS/Mac Catalyst](#tab/macios)
+### [Mac Catalyst](#tab/mac)
 
-No setup is required.
+Edit the `Info.plist` for `MacCatalyst` and add the following keys.
+```csharp
+    <key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoadsInMedia</key>
+		<true/>
+		<key>NSAllowsArbitraryLoads</key>
+		<true/>
+	</dict>
+	<key>UIBackgroundModes</key>
+	<array>
+		<string>bluetooth-central</string>
+		<string>audio</string>
+	</array>
+    <key>NSLocalNetworkUsageDescription</key>
+	<string></string>
```

Comment:
Is it just me or is the whitespace formatting misaligned here?

Reply:
Here's a screenshot of the info.plist from the updated sample app for reference:
<img width="639" alt="image" src="https://github.com/MicrosoftDocs/CommunityToolkit/assets/13558917/2b180bbd-b760-4bd0-b250-27dc2ec8be30">